docs/source/deployment/sql_server.rst

```diff
@@ -0,0 +1,212 @@
+Deploying to SQL Server Machine Learning Services
+=================================================
+
+Microsoft's Machine Learning Services is a feature in SQL Server that gives the ability to run Python and R scripts with relational data.
+We can use BentoML and other open-source packages, along with the Microsoft Python packages, for predictive analytics and machine learning. 
```

Comment:
A technical writing hint: It's better to avoid the first person ("we") because the reader can become confused. It's not clear who "we" is in this context.

Suggestion:
```
This guide demonstrates how to use BentoML along with the Microsoft ...
```